### PR TITLE
hw-mgmt: config: Fix PCI bus for SGX-H100 system

### PR DESCRIPTION
**Pull request by sholeksandr**
Fix issue in PCI bus for SGX-H100 system
Fix default values for sfp/module/gearbox counter

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management.sh [hunk 1]
```diff
@@ -1543,11 +1543,14 @@ create_symbolic_links()
...
 	fi
-	if [ ! -f "$thermal_path/module_counter" ]; then
-		echo 0 > "$thermal_path"/module_counter
+	if [ ! -f "$config_path/module_counter" ]; then
+		echo 0 > "$config_path"/module_counter
+	fi
+	if [ ! -f "$config_path/sfp_counter" ]; then
+		echo 0 > "$config_path"/sfp_counter
 	fi
 }
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management.sh:1556]: Why this code there? It was in different patch.

File: usr/usr/bin/hw-management.sh [hunk 2]
```diff
@@ -1601,7 +1604,7 @@ set_asic_pci_id()
 		echo "$asic2_pci_bus_id" > "$config_path"/asic2_pci_bus_id
 		echo 2 > "$config_path"/asic_num
 		;;
-	HI131|HI141|142)
+	HI131|HI141|HI142)
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management.sh:1607]: How I put it with no HI...

File: usr/usr/bin/hw-management.sh (hunk 1 above)
Comment (sholeksandr) [reply usr/usr/bin/hw-management.sh:1556]: In prrevious commit it was mistakly created in a wrong folder. Must be in "config/"

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**